CustomLogAppender/README.md

```diff
@@ -1,28 +1,26 @@
 # Custom Log Appender
-Demonstrates how you could write a Log Appender that can be enabled to be used by Axon Ivy. trough the `configurations/log4jconfig.xml` file.
+Demonstrates how you could write a Log Appender that can be enabled to be used by Axon Ivy.
+trough the `configurations/log4j2.xml` file. The appender attaches itself to desired logs.
+To see an output you need an error log in the project to which the appender can attach itself.
 
 ## Integration
 - The custom log appender must contribute the Axon Ivy core bundle `ch.ivyteam.util`, that contains Log4j and its related libraries. Therefore this bundle registers itself as buddy of `ch.ivyteam.util`. See [META-INF/MANIFEST.MF](https://github.com/ivy-samples/tomcatValve/blob/master/CustomLogAppender/META-INF/MANIFEST.MF) `Eclipse-RegisterBuddy: ch.ivyteam.util`
```

Comment:
this is the most important point of this demo: but the bundles are no longer correct; please update